code/game/objects/items/storage/bags.dm

```diff
@@ -133,24 +131,30 @@
 
 // Modified handle_item_insertion.  Would prefer not to, but...
 /obj/item/storage/bag/sheetsnatcher/handle_item_insertion(obj/item/item, prevent_warning = 0, mob/user)
+	if(!item || !istype(item, /obj/item/stack/sheet))
+		return FALSE
 	var/obj/item/stack/sheet/sheet = item
-	if(!sheet) return FALSE
-
 	var/amount
 	var/inserted = FALSE
 	var/current = 0
-	for(var/obj/item/stack/sheet/sheet_sub in contents)
+
+	for(var/content in contents)
+		if(!istype(content, /obj/item/stack/sheet))
```

Comment:
how can anything be in contents without being a stack/sheet? 
there's a check on the insertion, right?

Reply:
It's always better to be safe than sorry.